### ISSUE TITLE
Naming Conventions: Label as used in Data Elements and CodeSystems 

### ISSUE THREAD

**Issue by Dtphelan1** [standardhealth/shr-json-export#5]
Our conceptualization of label for CodeSystems seem sufficiently divorced from the use of label in DataElement . In the DataElement case, label seems synonymous with something like an shr_identifier, the thing you look up in the shr to get to that value. In the Codesystem Value Case, label seems synonymous with the display value, but separate from the code, which appears to me to be the closest thing to an shr_identifier.  Let me know if this disambiguation is clear at all and what we'd want to do, or if this is too hazy an explanation. For context: 

DataElement Value:
{
              "type": "DataElement",
              "label": "ActiveFlag",
              "isEntry": false,
              "concepts": [],
              "description": "If the ActiveFlag is false, it indicates the record is no longer to be used and should generally be hidden for the user in the UI.",
              "grammarVersion": { ... },
              "value": { ... }
              },
              "children": []
}

CodeSystem Value: 
{
              "label": "Preferred language (http://standardhealthrecord.org/fhir/CodeSystem/shr-actor-LanguageQualifierVS:preferred_spoken)",
              "type": "Concept",
              "system": "http://standardhealthrecord.org/fhir/CodeSystem/shr-actor-LanguageQualifierVS",
              "code": "preferred_spoken",
              "display": "Preferred language",
              "url": "http://standardhealthrecord.org/fhir/CodeSystem/shr-actor-LanguageQualifierVS"
            },

**Comment by gregquinn2001** [comment]
I agree they don't seem equal, but I think code system should be changed. The parenthetical part should be removed. The code system label should just be "Preferred language" probably.
